projects/elife.yaml

```diff
@@ -24,6 +24,10 @@ defaults:
         - https://github.com/elifesciences/builder-base-formula
     aws:
         account-id: 512686554592
+        # can this configuration be reused or is it unique? 
+        # typically it can be reused, but some configurations embed values that are unique to a named instance.
+        # for example: the 'prod' configuration for 'journal' *is* unique and cannot be reused.
```

Comment:
"for example: the 'prod' configuration for 'journal' is unique to `journal--prod` and *cannot* be reused."